frontend/src/pages/accounts/[id].tsx

```diff
@@ -14,10 +15,27 @@ import Content from "../../components/utils/Content";
 import TransactionIcon from "../../../public/static/images/icon-t-transactions.svg";
 
 import { Translate } from "react-localize-redux";
+import { NextPageContext } from "next";
 
-class AccountDetail extends Component {
-  static async getInitialProps({ req, query: { id }, res }) {
-    if (/[A-Z]/.test(id)) {
+interface Props extends PageNetworkProps {
+  account:
+    | Account
+    | {
+        accountId: string;
+      };
+  accountFetchingError?: unknown;
+  accountError?: unknown;
+}
+
+class AccountDetail extends Component<Props> {
+  static async getInitialProps({
+    req,
+    query: { id: rawId },
+    res,
+  }: NextPageContext) {
+    // Change to string[] if pagename changes from `[id]` to `[...id]`
```

Comment:
It this comment a little too obvious?

Reply:
Well, it is a bit hard to reason about this comment at first glance, but after a second thought it is obvious and next I thought of “why did you tell me this?”. We either need to rephrase it or just drop it (there is no meaningful use-case for […id])

Reply:
Let's remove it, actually